harpy/metassembly.py

```
@@ -69,29 +67,32 @@
         config.write(f"snakemake_log: {sm_log}\n")
         config.write(f"output_directory: {output_dir}\n")
         config.write(f"barcode_tag: {bx_tag.upper()}\n")
-        config.write(f"clusters: {clusters}\n")
-        config.write(f"contig_coverage: {contig_cov[0]},{contig_cov[1]}\n")
+        config.write("spades:\n")
+        config.write(f"    max_memory: {max_memory}\n")
+        if kmer_length == "auto":
+            config.write(f"    k: auto\n")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)